manifests/init.pp

```diff
@@ -497,7 +497,7 @@
   Variant[Undef,Integer,Pattern[/^(\d+)$/]] $rabbitmq_port = undef,
   Optional[String]   $rabbitmq_host = undef,
   Optional[String]   $rabbitmq_user = undef,
-  Optional[String]   $rabbitmq_password = undef,
+  Variant[Undef,Sensitive,String] $rabbitmq_password = undef,
```

Comment:
@ghoneycutt So, a working but not optimal solution would be to merge this pr without the tests, and leave them for later or to more experts hands.